service_container/parameters.rst

```diff
@@ -243,8 +243,8 @@ Constants as Parameters
 -----------------------
 
 Setting PHP constants as parameters is also supported.
-To take advantage of this feature, map the name of your constant to a parameter
-key and define the type as ``constant``.
+For the XML format, you need to map the name of your constant to a parameter
+key and define the type as ``constant``:
```

Comment:
Sorry, if that wasn't clear enough. I would replace the whole paragraph (line 245 to 247) with this:

> Setting PHP constants as parameters is also supported: